geojson/index.d.ts

```diff
@@ -18,116 +17,107 @@ declare namespace GeoJSON {
     /***
     * http://geojson.org/geojson-spec.html#positions
     */
-    export type Position = number[]
+    export type Position = number[];
 
     /***
     * http://geojson.org/geojson-spec.html#geometry-objects
     */
-    export interface GeometryObject extends GeoJsonObject
-    {
-        coordinates: any
+    export interface DirectGeometryObject extends GeoJsonObject {
```

Comment:
I think it is better to omit exporting, because it is for internal purpose and will pollute the type-definition. I'll commit it later, if there is no protest against...